src/wallet/wallet.cpp

```diff
@@ -3946,7 +3946,7 @@ bool DoMigration(CWallet& wallet, WalletContext& context, bilingual_str& error,
 
             DatabaseStatus status;
             std::vector<bilingual_str> warnings;
-            std::string wallet_name = wallet.Getname() + "_watchonly";
+            std::string wallet_name = wallet.GetName() + "_watchonly";
```

Comment:
Perhaps the change in `src/wallet/wallet.cpp` in https://github.com/bitcoin/bitcoin/commit/7ab26895d879e8cadd70109bd84c0b7e586c31a2 ('Add migratewallet RPC') could be squashed into https://github.com/bitcoin/bitcoin/commit/211c12bcd4545aa457a2efda17dfbf6a486407c3 ('Implement MigrateLegacyToDescriptor').

Currently, https://github.com/bitcoin/bitcoin/commit/211c12bcd4545aa457a2efda17dfbf6a486407c3 does not compile..

Reply:
Done